docs/file-reference.md

```diff
@@ -136,6 +137,7 @@ Each "app" (Kedge file) is a Kubernetes <a target="_blank" href="https://v1-6.do
 | Field    | Type     | Required     | Description  |
 |----------|----------|--------------|--------------|
 | name | string   | yes          | The name of the app or micro-service this particular file defines. |
+| appversion | string | no           | The version of the app or micro-service this particular file defines. |
```

Comment:
microservice is one word

Reply:
hi @cdrage,

If you check name and controller field in the same table, it is written like this, if you still want the change then I will do.

Thanks

Reply:
yeah, both need to be changed :+1:

Reply:
pushed a PR here: https://github.com/kedgeproject/kedge/pull/493